Core/Object Arts/Dolphin/Base/IntegerTest.cls

```diff
@@ -68,6 +68,14 @@ testIsPowerOfTwo
 	powersOfTwo do: [:each | self assert: each isPowerOfTwo].
 	nonPowersOfTwo do: [:each | self deny: each isPowerOfTwo]!
 
+testIsPrime
+	| isPrimeResults |
+
+	isPrimeResults := #( 1 2 3 4 5 6 7 8 9 10 11 12 13 1000 7919 ) collect: [ :num | num isPrime ].
```

Comment:
-2, -1, and 0 as well. Some large integer cases would be nice (although these will be increasingly slow)